monitoring/local_monitoring/vpa_demo/vpa_demo.sh

```diff
@@ -23,6 +23,10 @@ source ${current_dir}/../common.sh
 
 # Default docker image repo
 export KRUIZE_DOCKER_REPO="quay.io/kruize/autotune_operator"
+NAMESPACE="openshift-tuning"
```

Comment:
@shreyabiradar07 If the default cluster_type is kind, namespace default should be set to monitoring, you can change it later based on cluster_type

Reply:
Looks like NAMESPACE is not being used at all, can you remove it